src/convert.js

```diff
@@ -108,6 +110,14 @@ export function convert(data, columnDecoder) {
   if (ctype === 'INTERVAL') {
     throw new Error('parquet interval not supported')
   }
+  if (type === 'BYTE_ARRAY' && geospatial) {
+    const arr = new Array(data.length)
+    for (let i = 0; i < arr.length; i++) {
+      arr[i] = decodeWKB(data[i])
+    }
+    return arr
+    // TODO: use custom parquet parser
```

Comment:
yea we should add to parsers just like dates (also would be nice if we could specify custom string decoder)